### ISSUE TITLE
blurhash for placeholders?

### ISSUE THREAD

**Issue by shish** [calaldees/KaraKara#140]
https://github.com/woltapp/blurhash

Currently browser2 has a hardcoded inline-svg placeholder for when thumbnail images are loading - I wonder if computing blurhashes and including them in tracks.json would look nicer and render efficiently...

**Comment by shish** [comment]
Somebody made an even better version - https://evanw.github.io/thumbhash/

**Comment by calaldees** [comment]
Interesting concept

I came across this for css-only blur hashs
https://leanrada.com/notes/css-only-lqip/

I skilled the thumbhash page. With the `thumbhash`, what do we do with the binary/base64 data to get it into the web world? Is it just something like`<img src="data:image/avif;base64,xxx">`? Or is it plopped inside an svg or something?

**Comment by shish** [comment]
> (edit: AH! So I'm guessing there is a js blurhash library and data-blurhash= attr)

Yes that (Or React equivalent, but that's the same thing with different syntax)


> I came across this for css-only blur hashs

That does look pretty cool :O

I'd imagine that the tricky part is updating PM3 to calculate whichever hashes we want, all three of these options looks pretty trivial to display on the client side
